editoast/src/views/electrical_profiles.rs

```diff
@@ -41,10 +41,13 @@ async fn get(
     db_pool: Data<DbPool>,
     electrical_profile_set: Path<i64>,
 ) -> Result<Json<ElectricalProfileSetData>> {
-    let electrical_profile_set = electrical_profile_set.into_inner();
+    let electrical_profile_set_id = electrical_profile_set.into_inner();
     block(move || {
         let mut conn = db_pool.get()?;
-        let ep_set = ElectricalProfileSet::retrieve(&mut conn, electrical_profile_set)?;
+        let ep_set = ElectricalProfileSet::retrieve_conn(&mut conn, electrical_profile_set_id)?
```

Comment:
Here you can use `retrieve(db_pool, id).await?` in order to get rid of the `block`